khal/calendar_display.py

```diff
@@ -26,6 +26,8 @@
 
 from click import style
 
+from terminal import urwid_to_click
```

Comment:
you'll need to change this to ``from .terminal import urwid_to_click`` for python 3 compat